playground/mocks/idp/idx/data/factor-verification-webauthn.json

```diff
@@ -0,0 +1,94 @@
+{
+    "stateHandle": "01pCiYqxWzm-iaFdL2W26KHFB-myLMJk2JBrLWmPxF",
+    "version": "1.0.0",
+    "expiresAt": "2019-08-14T20:42:04.000Z",
+    "step": "AUTHENTICATE",
+    "intent": "LOGIN",
+    "remediation": {
```

Comment:
where to pass the challenge that is currently in _embedded 

`{
  "stateToken": "00MBvZeuINdrwaY5dRaJbgfhJLIq3B8zKos1iSPPy1",
  "expiresAt": "2019-10-15T21:45:38.000Z",
  "status": "MFA_CHALLENGE",
  "factorResult": "CHALLENGE",
  "factorType": "webauthn",
  "challengeType": "FACTOR_TYPE",
  "_embedded": {
    "user": {
      "id": "00u1gnxJRh6uwgVOB0g4",
      "passwordChanged": "2019-10-10T22:37:56.000Z",
      "profile": {
        "login": "administrator1@test.com",
        "firstName": "administrator1",
        "lastName": "administrator1",
        "locale": "en",
        "timeZone": "America/Los_Angeles"
      }
    },
    "factors": [
      {
        "id": "fwf1hldEttEJHtBwu0g4",
        "factorType": "webauthn",
        "provider": "FIDO",
        "vendorName": "FIDO",
        "profile": {
          "credentialId": "AUbARv8IFlp3Cj2zn74ciMYcZofWVR7OOOmtBra243ivC-zkaz8LKbs2oUADxkzpZQ0-yw8MyqobIo89hL9lpu1GpnsMNEabtqvtsT7K9RJG7GQ37K60e6GU5g4lSgcRYIGtbx-3",
          "authenticatorName": "MacBook Touch ID"
        },
        "_links": {
          "verify": {
            "href": "https://idxrain.okta1.com:80/api/v1/authn/factors/fwf1hldEttEJHtBwu0g4/verify",
            "hints": {
              "allow": [
                "POST"
              ]
            }
          }
        }
      }
    ],
    "policy": {
      "allowRememberDevice": false,
      "rememberDeviceLifetimeInMinutes": "0",
      "rememberDeviceByDefault": false,
      "factorsPolicyInfo": null
    },
    "challenge": {
      "challenge": "ErRRFnVouygoK4IKHPXc",
      "extensions": {}
    }
  },
  "_links": {
    "next": {
      "name": "verify",
      "href": "https://idxrain.okta1.com:80/api/v1/authn/factors/webauthn/verify",
      "hints": {
        "allow": [
          "POST"
        ]
      }
    },
    "prev": {
      "href": "https://idxrain.okta1.com:80/api/v1/authn/previous",
      "hints": {
        "allow": [
          "POST"
        ]
      }
    },
    "cancel": {
      "href": "https://idxrain.okta1.com:80/api/v1/authn/cancel",
      "hints": {
        "allow": [
          "POST"
        ]
      }
    }
  }
}`

Reply:
what does `challenge` mean?